spec/models/family_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Family do
+  describe "validations" do
+    subject { Family.new(last_name: nil) }
```

Comment:
I don't think we need the subject block here

Reply:
👍